src/zhinst/toolkit/exceptions.py

```diff
@@ -1,5 +1,14 @@
 """Global zhinst-toolkit exceptions."""
 
 
+class ToolkitError(RuntimeError):
+    """Base class for `zhinst.toolkit` errors.
+
+    .. versionadded:: 0.5.2
+    """
+
+    pass
+
+
```

Comment:
The `ValidationError` should derive from `ToolkitError`. This would not break any existing code since the `ValidationError` currently derives from `Exception` itself